app/controllers/shelters_controller.rb

```diff
@@ -23,6 +23,7 @@ def create
 
   def show
     @shelter = Shelter.find(params[:id])
+    @reviews = Review.all
```

Comment:
I think this should be: 
`@reviews = Review.where(shelter_id: @shelter.id)`
Otherwise, it will display all reviews for all shelters.